fracas/rules.cpp

```diff
@@ -248,6 +248,9 @@ static GameDefs g = {
 	1,	// WORLD_EVENTS
 	0,	//FACTION_STATISTICS
 	FactionActivityRules::DEFAULT,	// FACTION_ACTIVITY
+	DestroyBehavior::INSTANT,	// DESTROY_BEHAVIOR
+	200,	// MIN_DESTROY_POINTS,
+	34,	// MAX_DESTROY_PERCENT
```

Comment:
Why do we want to limit it to 34 instead of 100?

Reply:
In that particular ruleset it won't be used because instant destruction is turned on. I just copied the same values around.